server/prisma/migrations/20240215202648_init/migration.sql

```diff
@@ -405,3 +405,6 @@ ALTER TABLE "ticketrestrictions" ADD CONSTRAINT "ticketrestrictions_seasonticket
 
 -- Manually Add Refund Constraint
 ALTER TABLE "refunditems" ADD CONSTRAINT "refund_ticketitems_check" CHECK (("donationid_fk" is null AND "orderticketitemid_fk" is not null) OR ("orderticketitemid_fk" is null AND "donationid_fk" is not null));
+
+-- Make contactid_fk Nullable
+ALTER TABLE "orders" ALTER COLUMN "contactid_fk" DROP NOT NULL;
```

Comment:
This has to be in a new, distinct database migration file

Reply:
I created the new migration and moved this code out of the init migration.